sclblpy_tutorial.ipynb

```diff
@@ -0,0 +1,543 @@
+{
```

Comment:
It would be good to also add this (or some other) examples as .py files. That makes it easier to review them, and (for some, such as, for one, me) to run them.

Reply:
the .py files are out of scope for the current task (making the package useable again), but I'll add it to the monday tasks for later! (it should be really easy to just export the notebook as a .py file which I guess would be a good solution already :) )